src/webgpu/api/validation/encoding/cmds/copyTextureToTexture.spec.ts

```diff
@@ -20,14 +21,24 @@ class F extends ValidationTest {
     source: GPUImageCopyTexture,
     destination: GPUImageCopyTexture,
     copySize: GPUExtent3D,
-    isSuccess: boolean
+    isSuccess: boolean,
+    /** If submit is true, the validaton error is expected to come from the submit and encoding
+     * should succeed. */
+    submit?: boolean
```

Comment:
Yes, but existing code in this testsuit call this helper function (`TestCopyTextureToTexture`) a lot, changing it in this way by adding a optional parameter (rather than modifying `isSuccess` into a three values enum) ensure that we can leave the existing code unchanged. On the other hand, many other similar helper function do the same thing, e.g.  `testRun` in `api\validation\image_copy\image_copy.ts`.

Reply:
hm I think it's ok to change the existing code to improve the readability. That way it's more obvious if someone is reading it what the test is expecting to happen

Reply:
Done.

Reply:
Thanks, still LGTM!